WooCommerce/Classes/ViewRelated/Text View Screen/TextViewViewController.swift

```diff
@@ -1,5 +1,14 @@
 import UIKit
 
+protocol TextViewViewControllerDelegate: class {
+
+    // Text validation.
+    func shouldValidate(text: String) -> Bool
```

Comment:
nit suggestion: `should*` function name sounds like it's returning whether we should perform some action to me, not sure if `validate(text: String) -> Bool` would sound a bit more clear that it returns the validation result?